src/uct/ib/base/ib_iface.c

```diff
@@ -693,10 +693,17 @@ uct_ib_iface_roce_is_reachable(const uct_ib_device_gid_info_t *local_gid_info,
     remote_ib_addr_af = uct_ib_address_flags_get_roce_af(remote_ib_addr_flags);
     if ((uct_ib_device_roce_gid_to_sockaddr(local_ib_addr_af,
                                             &local_gid_info->gid,
-                                            &sa_local) != UCS_OK) ||
-        (uct_ib_device_roce_gid_to_sockaddr(remote_ib_addr_af,
-                                            remote_ib_addr + 1,
-                                            &sa_remote) != UCS_OK)) {
+                                            &sa_local) != UCS_OK)) {
+        uct_iface_fill_info_str_buf(
+               params, "Couldn't convert local RoCE address to socket address");
+        return 0;
+    }
+
+    if (uct_ib_device_roce_gid_to_sockaddr(remote_ib_addr_af,
+                                           remote_ib_addr + 1,
+                                           &sa_remote) != UCS_OK) {
+        uct_iface_fill_info_str_buf(
+               params, "Couldn't convert remote RoCE address to socket address");
```

Comment:
"Invalid remote RoCE v2 GID %s family %u"

Reply:
this message was merged from master